web/views.py

```
@@ -7018,3 +7021,97 @@ def users_list(request: HttpRequest) -> HttpResponse:
+                "upvotes": topic.upvote_count(),
+                "downvotes": topic.downvote_count(),
+                "score": topic.vote_score(),
+            }
+        )
+
+    except ForumTopic.DoesNotExist:
+        return JsonResponse({"error": "Topic not found"}, status=404)
+
+
+@login_required
+def reply_vote(request, pk):
```

Ruff (0.8.2)

Missing return type annotation for public function `reply_vote`

(ANN201)